server/pulp/plugins/loader/api.py

```diff
@@ -484,14 +485,35 @@ def _load_type_descriptors(path):
     return descriptors
 
 
-def _load_type_definitions(descriptors, drop_indices=False):
+def _load_type_definitions(definitions, drop_indices=False):
```

Comment:
I know this isn't your code, but it doesn't look to me like this function is actually loading anything. If in your work you figured out the role this plays, could you add some comments?